README.md

```diff
@@ -11,7 +11,8 @@ Try it in [Expo](https://expo.io/):
 * [Project structure](#project-structure)
 * [Working with GraphQL API](#working-with-graphql-api)
 * [Offline first](#offline-first)
-* [Best Practices](#best-practices)
+* [Error handling](#error-handling)
+* [Best practices](#best-practices)
```

Comment:
Capital H, capital P.

Reply:
Would you prefer first capitals everywhere? I did this change intentionally but I also prefer first capitals everywhere in English headings...

Reply:
Now that I tried to find some arguments, I'm not so sure anymore. There are two main standards - *title case* and *sentence case*. Until now I thought the title case is more "correct", but apparently the sentence case is also a valid option. Maybe we can skip this decision for now and continue using the sentence case.

https://en.wikipedia.org/wiki/Letter_case#Headings_and_publication_titles